packages/twenty-front/src/modules/activities/emails/components/Emails.tsx

```diff
@@ -28,11 +32,16 @@ export const Emails = () => (
       <StyledH1Title
         title={
           <>
-            Inbox <StyledEmailCount>2</StyledEmailCount>
+            Inbox <StyledEmailCount>{emails.length}</StyledEmailCount>
```

Comment:
the count is likely to be pulled separately as we have paginations on emails (and probably an infinite scroll). But OK for now + we don't know how to do aggregations on the backend yet :p